keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+// `----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+				[_NUMS] = KC_KEYMAP(
+// ,----+----+----+----+----+----.    ,----+----+----+----+----+----.
+								, 1  , 2  , 3  , 4  , 5  ,      6  , 7  , 8  , 9  ,SLSH,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								,CAPS, INS,HOME,PGUP,LPRN,     RPRN, 4  , 5  , 6  ,ASTR,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								,NLCK, DEL, END,PGDN,SCLN,     COMM, 1  , 2  , 3  ,MINS,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								,    ,    ,    ,    ,    ,         , 0  ,DOT ,EQL ,PLUS,
+// `----+----+----+----+----+----'    `----+----+----+----+----+----'
+								),
+
+				[_ADJUST] = KEYMAP(
+// ,-------+--------+--------+--------+--------+--------.   ,-------+--------+--------+--------+--------+--------.
+		   RESET , XXXXXXX,  KC_UP , XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_DEL , \
+// |-------+--------+--------+--------+--------+--------|   |-------+--------+--------+--------+--------+--------|
+			 _______, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, XXXXXXX,    BL_TOGG, BL_STEP, XXXXXXX, XXXXXXX, XXXXXXX, _______, \
+// |-------+--------+--------+--------+--------+--------|   |-------+--------+--------+--------+--------+--------|
+		  _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, \
+// |-------+--------+--------+--------+--------+--------|   |-------+--------+--------+--------+--------+--------|
+		  _______, _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______, _______  \
+// `-------+--------+--------+--------+--------+--------'   `-------+--------+--------+--------+--------+--------,
+		)
+
+/*
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer